Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/exchangemgr/InternalExchangeManager.java

```diff
@@ -123,7 +123,7 @@ public OrganizationType getOrganization(String hcid) throws ExchangeManagerExcep
 
     @Override
     public String getDefaultExchange() {
-        return null;
+        return null != exInfo ? exInfo.getDefaultExchange() : null;
```

Comment:
Rather than just a null check, shouldn't we attempt to load the exchange info file first?